src/core/MeganavItemsSignedIn.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import MeganavItemsSignedIn from "./MeganavItemsSignedIn";
+
+const meta = {
+  component: MeganavItemsSignedIn,
```

Comment:
Also, a wrapper around the component that imposes a maximum width would help to present this component in a way which is more visually recognisable to the actual Meganav (the flex behaviour means that the account name is way over on the left). 

Take a look at some of the other stories within this repo for examples of how `render: () => <>code here<>` works, that allows you to render the component whilst also modifying how it's presented.

Reply:
@jamiehenson I agree with your comment,  I did some updates
- I removed the `MeganavItemsSignedIn.stories` 
- updated `Meganav.stories.tsx` by adding `MeganavSignedIn`  component so the `Default` component will always be not logged-in and the `MeganavSignedIn` will always be SignedIn version. I updated the HTTP request with some dummy default values since the API seems not working yet?